include/dem/gear3_integrator.h

```diff
@@ -0,0 +1,71 @@
+/* ---------------------------------------------------------------------
+ *
+ * Copyright (C) 2019 - 2019 by the Lethe authors
+ *
+ * This file is part of the Lethe library
+ *
+ * The Lethe library is free software; you can use it, redistribute
+ * it, and/or modify it under the terms of the GNU Lesser General
+ * Public License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ * The full text of the license can be found in the file LICENSE at
+ * the top level of the Lethe distribution.
+ *
+ * ---------------------------------------------------------------------
+
+ *
+ * Author: Bruno Blais, Polytechnique Montreal, 2019
+ */
+
+#include <deal.II/particles/particle_handler.h>
+
+#include <dem/dem_solver_parameters.h>
+#include <dem/integrator.h>
+
+using namespace dealii;
+
+#ifndef gear3_integrator_h
+#  define gear3_integrator_h
+
+/**
+ * Implementation of a gear3 scheme for the integration
+ * of the particle motion. Note that reinitilization of force and torque is also
+ * integrated into integration class
+ *
+ * @note
```

Comment:
Can you write the formula of the integration method here and put a reference? It's not like explciit Euler which is trivial. We should also add the formula for the verlet integrator and explicite euler anyway.

Reply:
This is a third order method right?